src/MunicipalityRegistry.Api.Extract/Extracts/ExtractFiles/DbfFileWriter.cs

```diff
@@ -0,0 +1,37 @@
+namespace MunicipalityRegistry.Api.Extract.Extracts.ExtractFiles
+{
+    using System.IO;
+    using System.Text;
+    using Be.Vlaanderen.Basisregisters.Shaperon;
+
+    public class DbfFileWriter<TDbaseRecord> : ExtractFileWriter
+        where TDbaseRecord : DbaseRecord
+    {
+        private static Encoding Encoding => Encoding.GetEncoding(1252);
```

Comment:
Can we use `private static Encoding Encoding => DbaseCodePage.Western_European_ANSI.ToEncoding();` again for clarity instead of 1252?